codecov.yml

```diff
@@ -6,9 +6,24 @@ codecov:
   require_ci_to_pass: yes
 
 coverage:
+  range: "60...90"
   precision: 2
-  round: down
-  range: "70...100"
+  round: up
```

Comment:
Isn't `up` the exact opposite of what we want?
Our goal is e.g. for an exact coverage of `45.153123%` to be round to `45.15%`, right? According to the [docs](https://docs.codecov.io/docs/coverage-configuration) this is done using `round: down`.

Reply:
Depends on the way you look at it. You can look at Repository coverage or PR coverage.